src/app/pages/network/network-configuration-card/network-configuration-card.component.scss

```diff
@@ -0,0 +1,45 @@
+.mat-toolbar-row {
```

Comment:
Cards' height is not arranged when loading config and after loaded as before.
![Screenshot_17](https://user-images.githubusercontent.com/22980553/150561571-79d143e2-ba20-4f0b-8ac9-9a0d7d7a66d2.png)

Please try to add following style or something:
```
:host {
  display: block;
  flex: 1 1 1e-9px;

  .mat-card {
    height: calc(100% - 8px);
    max-width: 100%;
  }
}
```